css/style.css

```diff
@@ -46,13 +46,16 @@ html {
 
 #preview {
     position: absolute;
-    top: 85px;
+    float: left;
+    pointer-events: none;
+    opacity: 0.75;
+    /* top: 85px;

```

Comment:
recommend removing code instead of commenting it out; it'll be in git if we ever need it back, and the pull request UI will more clearly reflect your intent.

Reply:
Good point.